Dockerfile

```diff
@@ -1,16 +1,17 @@
+# Build SELinux packages for Arch linux in Docker/Podman container
+
 # Usage:
-#    sudo docker build -t arch-selinux-build .
-#
-# or with Podman on Arch Linux in July 2020:
-#    podman --cgroup-manager=cgroupfs build -t arch-selinux-build .
+#    docker build -t arch-selinux-build .
+# or   
+#    sudo podman build -t arch-selinux-build .
```

Comment:
Why do you need `sudo` to use `podman`? The main point of using `podman` is to be able to use containers from a non-privileged user.

Reply:
My podman configurations are not correct, sorry. Totally unnecessary modification from me.